Example/WalletApp/PresentationLayer/Wallet/Notifications/NotificationsInteractor.swift

```diff
@@ -32,7 +33,32 @@ final class NotificationsInteractor {
     }
 
     func subscribe(domain: String) async throws {
-        try await Notify.instance.subscribe(appDomain: domain, account: importAccount.account)
+        return try await withCheckedThrowingContinuation { continuation in
+            var cancellable: AnyCancellable?
+            cancellable = subscriptionsPublisher
+                .setFailureType(to: Error.self)
+                .timeout(10, scheduler: RunLoop.main, customError: { Errors.subscribeTimeout })
```

Comment:
I am not sure it's a good idea to handle it this way here with a subscriptionsPublisher and timeout.
potentially it's possible that Notify Server will create a subscription and subscribe function here will throw.

Reply:
I moved it outside of the sdk, into NotificationsInteractor. So it's just sample app thing right now. For that moment it was works fine, let's keep it and  watch how it goes

Reply:
In sample app I think it’s good
Probably not the best integration example tho